pkg/hub/manager.go

```diff
@@ -185,12 +185,13 @@ func (o *AddOnOptions) RunControllerManager(ctx context.Context, controllerConte
 		controllerContext.EventRecorder,
 	)
 
-	go clusterInformers.Start(ctx.Done())
-	go workInformers.Start(ctx.Done())
-	go kubeInformers.Start(ctx.Done())
-	go configInformers.Start(ctx.Done())
-	go apiExtensionsInformers.Start(ctx.Done())
-	go addOnInformers.Start(ctx.Done())
+	clusterInformers.Start(ctx.Done())
+	workInformers.Start(ctx.Done())
+	kubeInformers.Start(ctx.Done())
+	configInformers.Start(ctx.Done())
+	apiExtensionsInformers.Start(ctx.Done())
+	addOnInformers.Start(ctx.Done())
+
```

Comment:
The commit message associated with this change does not talk about why we stopped using goroutines. 
Is it because the SharedInformerFactory.Start() method internally uses goroutines? If so, can you include it in the commit message.

Reply:
> Is it because the SharedInformerFactory.Start() method internally uses goroutines? 

yes. I guess I threw in that change w/o mentioning it :)